fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/data/ElectionHandler.java

```diff
@@ -39,7 +41,7 @@ public ElectionHandler(
   /**
    * Process an ElectionSetup message.
    *
-   * @param context the HandlerContext of the message
+   * @param context       the HandlerContext of the message
```

Comment:
There's an empty tab space here

Reply:
I see that my auto format does this for some reason to align it with the parameter of the next line😕 . I'll revert and disable it.

Reply:
There were some other formatting issues too because I was using another laptop and forgot to tick enable google-java-format 🙃. But it's fixed now.